notebooks/contrib-dev/Filtration.ipynb

```diff
@@ -1,1030 +1,1316 @@
+        "\n",
+        "For the same pressure drop, determine the size of the filter to process 4000 L of cell suspension in 20 minutes.\n",
+        "\n",
+        "> **Hint**: Use the Ruth equation to calculate the required area using:\n",
+        "$V^{2} + 2VV_{0} = Kt $\n",
+        "\n",
+        "where $K = (\\frac{2A^{2}}{𝛼C𝜇})Δpg_{c}$ and $V_{0} = \\frac{r_{m}}{𝛼C}A$.\n",
+        "\n",
+        "**Record your answer using paper and pencil.**\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "DimdzD-4RY1g"
+      },
+      "source": [
+        "Use your values of $V$, $ΔP$, and $t$ to determine the size of the filter. <br>\n",
+        ">**Hint**: Reformat the Ruth equation to be in terms of surface area only. This should produce a quadratic function. Solve the quadratic equation.\n",
+        "\n",
+        "**Pencil and Paper Solution:**\n",
+        "\n",
+        "$$\\left[\\left(\\frac{2A^{2}}{𝛼C𝜇}\\right)Δpg_{c}\\right](t) - 2V\\left(\\frac{r_{m}}{𝛼C}A\\right) -V^2 = 0$$\n",
+        "\n",
+        "$$3.59A^2-5.54A -8 = 0$$\n",
+        "\n",
+        "$$A = 2.45 m^2$$"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "t2VxSGC3T_nA"
+      },
+      "source": [
+        "Let's use Python as a calculator to check our answer."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "7CM9RSB7IaDU",
+        "outputId": "95cccb58-1d8e-4b00-cd85-9c90c77f9922"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "The possible areas of the filter are: 2.4519924278922605 -0.9088169404270792 m^2\n"
+          ]
+        }
+      ],
+      "source": [
+        "### BEGIN SOLUTION\n",
+        "#Use quadratic equation to solve for the roots\n",
+        "root1 = (5.54+math.sqrt((5.54**2)-(4*3.59*-8)))/(2*3.59)\n",
+        "root2 = (5.54-math.sqrt((5.54**2)-(4*3.59*-8)))/(2*3.59)\n",
+        "### END SOLUTION\n",
+        "\n",
+        "print(\"The possible areas of the filter are:\", root1,root2,\"m^2\")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "z-3Pw57o2H0t"
+      },
+      "source": [
+        "Only one root is positive, and since we cannot have a negative area, the positive root must be the solution."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "HbJHoaXtwNJI"
+      },
+      "source": [
+        "#4. Pressure Drop Optimization\n",
+        "\n",
+        "Lets say we wanted to reduce the pressure drop in the system by changing the flow rate of the filtrate. Inuitively, we can predict that an increase in velocity of the filtrate across the filter will increase the pressure drop - but by how much?\n",
```

Comment:
Add a sentence explaining this notebook will compare three optimization approaches.